csaf_2.0/guidance/sbom-matching-interface.md

```diff
@@ -0,0 +1,91 @@
+## SBOM Matching Interface
+
+This documents serves as a guidance how to implement an interface for a [CSAF SBOM matching matching system](https://docs.oasis-open.org/csaf/csaf/v2.0/os/csaf-v2.0-os.html#9117-conformance-clause-17-csaf-sbom-matching-system).
```

Comment:
Let us be specific here: "the" instead of "an". The term "for" to me is also ambiguous  and I assume it is the interface between SBOM data and CSAF data. Question: Isn't "that" already the "matching system"? Sorry, in case I got that wrong.